README.md

```diff
@@ -72,3 +75,133 @@ services:
+
+### smartctl output vs smartctl_exporter output
+
+The S.M.A.R.T. attributes are mapped in
+[smartctl.go](https://github.com/prometheus-community/smartctl_exporter/blob/master/smartctl.go).
+Each function has a `prometheus.MustNewConstMetric` or similar function with the
+first parameter being the metric name. Find the metric name in
+[metrics.go](https://github.com/prometheus-community/smartctl_exporter/blob/master/metrics.go)
+to see how the exporter displays the information. This may sound technical, but
+it's crucial for understanding how data flows from `smartctl` to
+`smartctl_exporter` to Prometheus.
+
+If the data looks incorrect, check the
+[Smartmontools Frequently Asked Questions (FAQ)](https://www.smartmontools.org/wiki/FAQ).
+It's likely your question may already have an answer. If you still have
+questions, open an [issue]().
+
+## Gathering smartctl data
+Follow these steps to gather smartctl data for troubleshooting purposes. If you
+have unique drives/data/edge cases and would like to "donate" the data, open a
+PR with the redacted JSON files.
+
+1. Run `scripts/collect-smartctl-json.sh` to export all drives to a
+   `smartctl-data` directory (created in the current directory).
+2. Run `scripts/redact_fake_json.py` to redact sensitive data.
+3. Provide the JSON file for the drive in question.
+
+```bash
+cd scripts
+./collect-smartctl-json.sh
+./redact-fake-json.py smartctl-data/*.json
+```
+
+## Run smartctl_exporter using JSON data
+The `smartctl_exporter` can be run using local JSON data. The device names are
+pulled from actual devices in the machine while the data is redirected to the
+`debug` directory. Save the JSON data in the `debug` directory using the actual
+device names using a 1:1 ratio. If you have 3 devices, `sda`, `sdb` and `sdc`,
+the `smartctl_exporter` will expect 3 files: `debug/sda.json`, `debug/sdb.json`
+and `debug/sdc.json`.
+
+Once the "fake devices" (JSON files) are in place, run the exporter passing the
+hidden `--smartctl.fake-data` switch on the command line. The port is specified
+to prevent conflicts with an existing `smartctl_exporter` on the default port.
+
+```bash
+smartctl_exporter --web.listen-address 127.0.0.1:19633 --smartctl.fake-data
+```
+
+# FAQ
+## How do I run `smartctl_exporter` against a JSON file?
+
+If you're helping someone else, request the output of the `smartctl` command
+above. Feed this into the `smartctl_exporter` using the
+hidden `--smartctl.fake-data` flag. If a `smartctl_exporter` is already running,
+use a different port; in this case, it's `19633`. Run `collect_fake_json.sh`
+first to collect the JSON files for **your** devices. Copy the requested JSON
+file into one of the fake files. After starting the exporter, you can query it
+to see the data generated.
+
+```bash
+# Dump the JSON files for your devices into debug/
+./collect_fake_json.sh
+
+# copy the test JSON into one of the files in debug/
+#cat > debug/sda.json
```

Comment:
Typo?

```suggestion
cat > debug/sda.json
```

Reply:
Fixed the command to indicate they need to copy the file(s) created from `collect_fake_json.sh`.